packages/datagateway-common/package.json

```diff
@@ -45,8 +45,10 @@
   "devDependencies": {
     "@mui/icons-material": "5.5.0",
     "@mui/material": "5.5.0",
+    "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "12.1.3",
     "@testing-library/react-hooks": "8.0.1",
+    "@testing-library/user-event": "^14.4.1",
```

Comment:
Could these dependencies be pinned? Worth doing because renovate will make a whole PR just to do this anyway if this were to get merged like this

Reply:
Yep, will fix.